static/usage/alert/presenting/controller/demo.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Alert</title>
+  <link rel="stylesheet" href="../../../common.css" />
+  <script src="../../../common.js"></script>
+  <script type="module" src="https://cdn.jsdelivr.net/npm/@ionic/core@6/dist/ionic/ionic.esm.js"></script>
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@ionic/core@6/css/ionic.bundle.css" />
+
+  <style>
+    .container {
+      flex-direction: column;
+    }
+  </style>
+</head>
+
+<body>
+  <ion-app>
+    <ion-content>
+      <div class="container">
+        <ion-button>Click Me</ion-button>
+        <p></p>
+      </div>
+    </ion-content>
+  </ion-app>
+
+  <script>
+    const output = document.querySelector('p');
+
+    async function presentAlert() {
+      const alert = document.createElement('ion-alert');
+      alert.header = 'Alert';
+      alert.subHeader = 'Important message';
+      alert.message = 'This is an alert!';
+      alert.buttons = ['OK'];
+
+      document.body.appendChild(alert);
+      await alert.present();
+
+      const { role } = await alert.onDidDismiss();
+      output.innerText = `Alert dismissed with role: ${role}`;
```

Comment:
I feel like we should not show the role in these examples, since it will always be `undefined`.

Reply:
You can also get it to show `backdrop` if you click that. I could add a role to the OK button if you'd prefer?

Reply:
It might be worthwhile to make this a destroy/cancel alert where each button has a special role: https://ionicframework.com/docs/api/modal

Reply:
Hmm, I think I'll just remove the role display from this basic example, and add it to the Buttons playground instead. That would make for a better separation of concerns and avoid the `undefined` role showing up.